bin/alluxio-secondary-masters.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
I thought of using secondary but eventually picked secondary-master because secondary might be also imply secondary worker though we don't have one. I am also slightly in favor of "secondary" over "secondary-master" since it is shorter. I would like to make sure Calvin and Gene are also fine with "secondary" before making this change. @gpang @calvinjia  How do you think?